manifests/init.pp

```diff
@@ -30,6 +30,7 @@
   Integer[0,1] $cvmfs_yum_testing_enabled                           = 0,
   Integer[0,1] $cvmfs_yum_gpgcheck                                  = 1,
   Variant[Stdlib::Filesource,Stdlib::Httpurl] $cvmfs_yum_gpgkey  = 'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CernVM',
+  Variant[Enum['absent'], String] $cvmfs_yum_includepkgs         = 'cvmfs,cvmfs-keys,cvmfs-server,cvmfs-config-default',
```

Comment:
Hi, 
I think this would be better as an array and then joined before passing to yumrepo.
Also since touching this while `,`s do work according to the yum docs the delimiters are actually meant to be spaces.

Reply:
Hi, I fully agree!
Give me a second, the commit I just pushed proves faulty in testing it.

Reply:
Done. This should be better :wink:.